docs/agent/agent.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[endpoint-agent]]
+= Endpoint agent
+
+Through the Elastic Endpoint agent, the endpoint sensor instantly detects and stops privilege escalation, defense evasion,
```

Comment:
I wrote, "Elastic Endpoint Agent" since the app can't be accessed through the Elastic agent as of this writing. Maybe we should adjust this to Endgame specific language?

Reply:
This language is all still being worked out - the security capabilities will be deployed via Elastic Agent. Right now the thinking is that we want to abstract away the concept of deploying something else - but just enabling the security capabilities of the Elastic agent.